### ISSUE TITLE
Date Picker - Selected date does not meet non-text contrast

### ISSUE THREAD

**Issue by betrue-final-final** [OfficeDev/office-ui-fabric-react#8918]
The selected state does not have 3:1 contrast compared to its unselected state

**Comment by atneik** [comment]
@betrue-final-final Just to be sure you mean the user selected date i.e. with lighter blue background (#c7e0f4) and the white (Friday the 3rd), and not the current date (Thursday the 2nd)?

![image](https://user-images.githubusercontent.com/785361/57093480-dc7ac780-6cc2-11e9-96f9-fc20af66ffad.png)
What should the new treatment be?

**Comment by betrue-final-final** [comment]
This actually is not a bug as the current day marker is smaller than the hover or selected state. It would work better if it was a different shape, but this technically passes.
